addons/l10n_account_withholding/views/account_tax_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_tax_form" model="ir.ui.view">
+        <field name="name">account.tax.form</field>
+        <field name="model">account.tax</field>
+        <field name="inherit_id" ref="account.view_tax_form"/>
+        <field name="arch" type="xml">
+            <field name="hide_tax_exigibility" position="after">
+                <!-- Hide for argentina, it should be handled by their own module -->
+                <field name="is_withholding_tax_on_payment" invisible="amount_type == 'group' or type_tax_use not in ['sale', 'purchase'] or country_code == 'AR'"/>
+                <field name="withholding_sequence_id" context="{'default_name': name}" invisible="amount_type == 'group' or not (type_tax_use == 'purchase' and is_withholding_tax_on_payment) or country_code == 'AR'"/>
```

Comment:
```suggestion
                <field name="is_withholding_tax_on_payment" invisible="amount_type == 'group' or type_tax_use not in ['sale', 'purchase'] or country_code == 'AR'"/>
                <field name="withholding_sequence_id" context="{'default_name': name}" invisible="amount_type == 'group' or not (type_tax_use == 'purchase' and is_withholding_tax_on_payment) or country_code == 'AR'"/>
```
why group? It should work only for percentage taxes I think
Also, plz indent the attributes

Reply:
Yes we may want to limit to percentage taxes, we can check with @tsb-odoo